docs/user/tools/using_secureboot_audit.md

```diff
@@ -0,0 +1,116 @@
+# SecureBootReport.py
```

Comment:
This first header is what shows as the tool name in the auto generated docs. This should be changed to Secureboot Report Tool (or something to that effect)